cloud/pkg/cloudhub/handler/messagehandler.go

```diff
@@ -145,26 +145,31 @@ func (mh *MessageHandle) OnRegister(connection conn.Connection) {
 	}
 
 	io := &hubio.JSONIO{Connection: connection}
+
+	if _, ok := mh.nodeRegistered.Load(nodeID); ok {
+		mh.nodeConns.Store(nodeID, io)
```

Comment:
Will old broken io be closed automatically?

Reply:
Done, already added the close func here.

Reply:
!!The stored value of nodeRegisted is not a connection, according to `mh.nodeRegistered.Store(info.NodeID, true)`